components/studentPortal/freemium/FreemiumDialogueComponent.tsx

```diff
@@ -20,8 +20,8 @@ export enum ModalStage {
 export interface FreemiumDialogComponentProps {
   trigger: boolean;
   triggerTitle?: string;
-  onClose: () => void;
-  startOnUpgradeModal: boolean;
+  onClose?: () => void;
```

Comment:
That only has to do with having the dialog box remain open when it is called from the tooltip so that when you move your cursor out of the tooltip it doesn't automatically close without you seeing the modal.  It is not needed for the first automatic onboarding.